pkg/syncer/metadatasyncer.go

```diff
@@ -831,6 +839,318 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 	return nil
 }
 
+func initStorageQuotaPeriodicSync(ctx context.Context) {
+	log := logger.GetLogger(ctx).WithOptions()
+	// create storagequotaperiodicsync CR
+	restConfig, err := config.GetConfig()
+	if err != nil {
+		log.Errorf("failed to get Kubernetes config. Err: %+v", err)
+		return
+	}
+	cnsOperatorClient, err := k8s.NewClientForGroup(ctx, restConfig, cnsoperatorv1alpha1.GroupName)
+	if err != nil {
+		log.Errorf("Failed to create CnsOperator client. Err: %+v", err)
+		return
+	}
+	sqPeriodicSync := &sqperiodicsyncv1alpha1.StorageQuotaPeriodicSync{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      StorageQuotaPeriodicSyncInstanceName,
+			Namespace: common.GetCSINamespace(),
+		},
+		Spec: sqperiodicsyncv1alpha1.StorageQuotaPeriodicSyncSpec{
+			SyncIntervalInMinutes: 10,
+		},
+	}
+	if err := cnsOperatorClient.Create(ctx, sqPeriodicSync); err != nil {
```

Comment:
I believe there should one CR, and it's updated, in this every syncer restart will create one